packages/compiler/src/init/init.ts

```diff
@@ -153,8 +153,10 @@ export async function initTypeSpecProject(
     },
   ]);
 
-  const libraries = await selectLibraries(template);
-  const parameters = await promptCustomParameters(template);
+  const [libraries, parameters] = await Promise.all([
```

Comment:
This one we do need fore sure to be done in sequence it prompts the user.

Reply:
> This one we do need fore sure to be done in sequence it prompts the user.

Done 🤝